public/app/features/variables/state/actions.ts

```diff
@@ -526,6 +527,7 @@ export const onTimeRangeUpdated = (
   try {
     await Promise.all(promises);
     const dashboard = getState().dashboard.getModel();
+    dashboard?.changeAffectsAllPanels();
     dashboard?.startRefresh();
```

Comment:
wonder if this makes more sense as param to startRefresh ? like dashboard.startRefresh({ changeAffectsAllPanels: true }) ;

Reply:
Ok, so then the caller is responsible for checking isPanelInEdit || isPanelInView? I wanted to separate those?

Reply:
Keeping this for now